circle.yml

```diff
@@ -0,0 +1,17 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: node:7.4.0
```

Comment:
Need node 8.1.x. unless this is an eslint specific requirement for 7.4.0

Reply:
@sam6321 Because of the issues present in the newer NPM, I do not recommend using the newer Node.